### DISCUSSION TITLE
Could not find package symfony/skeleton with version 5.4

### DISCUSSION THREAD

**Discussion by gearskullguy** [symfony/symfony#47327] [Q&A]
Maybe this is just a docs problem, but on page https://symfony.com/doc/5.4/setup.html#creating-symfony-applications (Creating Symfony Applications), upon running the command to install Symfony 5.4 with composer - `composer create-project symfony/skeleton:"^5.4" my_project_directory` - I get this error:

 [InvalidArgumentException]
  Could not find package symfony/skeleton with version 5.4.

I literally copied and pasted the command.

However, after some research, for whatever reason it works when appending `.*` to the `5.4` part... like this:

`composer create-project symfony/skeleton:"^5.4.*" my_project_directory`

Why does it only work the 2nd way, or are the docs wrong?

I'm using Windows 10 Pro 64-bit with XAMPP with PHP 8.1.2 and composer 2.2.3 by the way.

**Comment by fabpot** [comment]
I've just tried and it works for me with your exact same command (but I'm on Linux).

**Comment by cedrickabongo** [comment]
i am having the same issue with 6.4.15 while it works well with 6.4. it seems it does not like a minor version as the option because it will be looking for symfony/skeleton with the same minor version which does not exist.
